easybuild/easyconfigs/x/xtb/xtb-6.7.0-gfbf-2023a.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'MesonNinja'
+
+name = 'xtb'
+version = '6.7.0'
+
+homepage = 'https://xtb-docs.readthedocs.io'
+description = """ xtb - An extended tight-binding semi-empirical program package. """
+
+toolchain = {'name': 'gfbf', 'version': '2023a'}
+
+github_account = 'grimme-lab'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}]
+checksums = ['9cf1997064d2d5bde7fae4cec6f873469602e6554872ad79de4079f022855ae2']
+
+builddependencies = [
+    ('Meson', '1.1.1'),
+    ('Ninja', '1.11.1'),
+    ('pkgconf', '1.9.5'),
+]
+
+configopts = "-Dlapack='custom' "
+configopts += "-Dcustom_libraries='flexiblas' "
+configopts += "--buildtype release "
+
+runtest = 'meson'
+# Ensure test don't timeout and only use one process
+testopts = 'test -C %(builddir)s/easybuild_obj -t 100  --num-processes=1'
```

Comment:
I think the main issue is that on nodes with a lot of cores the tests use too many threads, in that case

```python
runtest = 'meson'
pretestopts = 'export OMP_NUM_THREADS=2 && '
testopts = 'test -C %(builddir)s/easybuild_obj -t 60'  # Ensure test don't timeout
```

for example, may be a better option (?)